examples/custom-elements/index.js

```diff
@@ -1,7 +1,8 @@
 import createVirtualEnvironment from '@locker/near-membrane-dom';
 
-const evalScript = createVirtualEnvironment();
-
+const evalScript = createVirtualEnvironment(window, {
+    globalThis: window,
```

Comment:
this is the default behavior anyways.

Reply:
After reading your comment [above](https://github.com/salesforce/near-membrane/pull/205#discussion_r721556000), about making things more explicit... I think we should further refine `createVirtualEnvironment`'s API to: 


```js
BrowserEnvironmentOptions {
    distortionCallback?: DistortionCallback;
    endowments?: object;
    keepAlive?: boolean;
    support?: SupportFlagsObject;
}

createVirtualEnvironment(
  globalObjectShape: object, 
  globalThis?: WindowProxy & typeof globalThis, 
  providedOptions?: BrowserEnvironmentOptions
)
```